job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/CloverPhpContext.groovy

```diff
@@ -0,0 +1,106 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.Context
+import javaposse.jobdsl.dsl.ContextHelper
+import javaposse.jobdsl.dsl.DslContext
+
+class CloverPhpContext implements Context {
+
+    Boolean publishHtmlReport = false
```

Comment:
use `boolean` instead of `Boolean`. Allowing `null` values does not make sense here.